apps/system/js/notifications.js

```diff
@@ -431,7 +431,7 @@ var NotificationScreen = {
     notificationNode.dataset.notificationId = detail.id;
     notificationNode.dataset.noClear = behavior.noclear ? 'true' : 'false';
     notificationNode.lang = detail.lang;
-    notificationNode.dataset.predefinedDir = detail.bidi;
```

Comment:
Make sure you also change the `detail.bidi` below:
https://github.com/mozilla-b2g/gaia/pull/29390/files#diff-9a74bae315146783bc7b24d5d7302a95R493